config/locales/google_tag_manager/es.yml

```diff
@@ -0,0 +1,12 @@
+es:
+  activemodel:
+    attributes:
+      organization:
+        entity_code: Código de ente
+        entity_name: Nombre de ente
```

Comment:
Maybe `entidad` is better, what do you think?

Reply:
done!